packages/cashier/tsconfig.json

```diff
@@ -2,19 +2,19 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "paths": {
-            "Components/*": ["components/*"],
-            "Config/*": ["config/*"],
-            "Constants/*":["constants/*"],
-            "Containers/*": ["containers/*"],
-            "Images/*": ["public/images/*"],
-            "Pages/*": ["pages/*"],
-            "Stores/*": ["stores/*"],
-            "Types": ["types"],
-            "Utils/*": ["utils/*"],
+            "Components/*": ["src/components/*"],
+            "Config/*": ["src/config/*"],
+            "Constants/*": ["src/constants/*"],
+            "Containers/*": ["src/containers/*"],
+            "Images/*": ["src/public/images/*"],
```

Comment:
@shayan-fs could u pls take a look at these folders, some of them don't need paths like the images folder, because we don't have any TS files inside them, so you can remove them

Reply:
sure. you're right. I forgot to remove them.